handbook/product/README.md

```diff
@@ -1,25 +1,23 @@
 # Product
 
-⚗️ Backlog: https://app.zenhub.com/workspaces/-product-backlog-coming-soon-6192dd66ea2562000faea25c/board
+Contributors in Fleet's product department [prioritize](#prioritizing-improvements) and [define](https://fleetdm.com/handbook/company/product-groups#drafting) the [changes we make to the product](https://fleetdm.com/handbook/company/product-groups#making-changes).
 
-## Job to be done
+Changes begin as [ideas](#intake) or [code](#outside-contributions) that can be contributed by anyone.
```

Comment:
Hey @zhumo, this diff is possibly harder to read than seeing the actual live content, so lemme include a peek at what that looks like, for convenience: 
![image](https://user-images.githubusercontent.com/618009/226821529-abfef192-dc9a-4076-8d68-d4dbd2ed2261.png)